forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -83,17 +93,13 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Checks the balance of a given PaymentMethod using a ForagePINEditText
+     * Checks the balance of a previously created [`PaymentMethod`](https://docs.joinforage.app/reference/payment-methods) via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
```

Comment:
To address the issue involving the lack of newlines. What are your thoughts on the following list item-based workaround for the submit methods in this file?

Also:

- can we wrap long lines to fit roughly inside the right-hand vertical line in Android studio (around 100 character line length limit)
  - the comments should be digestible for clients reading the comments inside Android Studio, and not just the docs site.
- can we use the notation `@param <parameter-name-not-type-here>` Description with reference to type of parameter here (as seen in the example below). Sticking with the KDoc standard should reduce the risk of our Dokka breaking and should provide a more familiar + consistent format throughout our comments. 
  

```
    /**
     * Checks the balance of a previously created [`PaymentMethod`](https://docs.joinforage.app/reference/payment-methods)
     * via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
     *
     * * On success, the response includes `snap` and `cash` fields that indicate the EBT Card's current SNAP and EBT Cash balances.
     * * On failure, for example in the case of [`ebt_error_14`](https://docs.joinforage.app/reference/errors#ebt_error_14), 
     * the response includes a list of [ForageError][com.joinforage.forage.android.network.model.ForageError] objects that you can unpack to troubleshoot the issue.
     *
     * @param params A [CheckBalanceParams][com.joinforage.forage.android.network.model] model
     * that passes a [`foragePinEditText`][com.joinforage.forage.android.ui.ForagePINEditText] instance
     * and a `paymentMethodRef`, found in the response from a call to [tokenizeEBTCard] or the [Create a `PaymentMethod`](https://docs.joinforage.app/reference/create-payment-method) endpoint.
     * @throws [ForageConfigNotSetException] If the [ForageConfig] is not set for the provided `foragePinEditText`.
     * @see * [SDK errors](https://docs.joinforage.app/reference/errors#sdk-errors) for more information on error handling.
     * * [Test EBT Cards](https://docs.joinforage.app/docs/test-ebt-cards#balance-inquiry-exceptions) to trigger balance inquiry exceptions during testing.
     * @return A [ForageApiResponse] object.
     */
```

<img width="700" alt="image" src="https://github.com/teamforage/forage-android-sdk/assets/32694765/76944df2-ccd8-4ef1-a119-437c7e8522fc">